packages/vscode-dependencies-validation/test/npmIssuesActionProvider.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { expect } from "chai";
+import * as proxyquire from "proxyquire";
+import type { CodeAction, CodeActionKind } from "vscode";
+import { NPMIssuesActionProvider } from "../src/npmIssuesActionProvider";
+import { range, textDocument, context, token, kind } from "./vscodeMocks";
+
+const diagnosticsProxy = {
```

Comment:
these two proxies are only used in the single test here.
I think they should be initialized inside the describe, otherwise future tests added in this file
could cease being self contained.

Reply:
although maybe as they only implement "reject" its fine

Reply:
ok